src/transforms/filter.rs

```diff
@@ -128,6 +129,10 @@ mod test {
             tx.send(log.clone()).await.unwrap();
 
             log.set_source_id(Arc::new(OutputId::from("in")));
+            log.set_upstream_id(Arc::new(OutputId::from("transform")));
+            log.metadata_mut()
+                .set_schema_definition(&Arc::new(Definition::default_legacy_namespace()));
```

Comment:
It could make it shorter, but I don't think that would help write tests very much. The hard part was determining which schema definition to use and which upstream id to pick, which a helper function won't really help with.